pay-api/src/pay_api/models/account_fee.py

```diff
@@ -70,6 +70,6 @@ class Meta:  # pylint: disable=too-few-public-methods
         """Returns all the fields from the SQLAlchemy class."""
 
         model = AccountFee
-        exclude = ['service_fee']
+        exclude = ['versions', 'service_fee']
```

Comment:
We included versions in auth, and it would query the versions table. I've opted to just put them in each schema instead of BaseSchema here because there are way less instances of them.